core/src/smartcontracts/isi/triggers/set.rs

```diff
@@ -539,6 +540,42 @@ pub enum LoadedExecutable {
     Instructions(Vec<InstructionBox>),
 }
 
+impl<F: Clone> From<&Action<F, LoadedExecutable>> for SerialAction<F> {
+    fn from(r: &Action<F, LoadedExecutable>) -> Self {
+        let exec = match &r.executable {
+            LoadedExecutable::Wasm(lw) => Executable::Wasm(WasmSmartContract(
+                lw.module
+                    .serialize()
+                    .expect("This failing means iroha was compiled incorrectly"),
```

Comment:
Not Iroha. Trigger -- is being writte by external users. Take a look at another place in this file where `serialize()` is used

Reply:
Well, technically both could cause this. However, you shouldn't lead peoeple down a single path if there are multiple causes. It doesn't look to me that this is an error with a single unambiguous cause.